oauthproxy.go

```diff
@@ -148,6 +148,19 @@ func (u *UpstreamProxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // servers
 func NewReverseProxy(target *url.URL, opts *options.Options) (proxy *httputil.ReverseProxy) {
 	proxy = httputil.NewSingleHostReverseProxy(target)
+	// Settings from net/http DefaultTransport except timeout
```

Comment:
Can you add a link to the code for this?

Reply:
Added link to comment.

Reply:
Link added.